Source/Scene/Cesium3DTileset.js

```diff
@@ -1468,8 +1527,23 @@ define([
      * @exception {DeveloperError} The tileset must be 3D Tiles version 0.0.  See https://github.com/AnalyticalGraphicsInc/3d-tiles#spec-status
      */
     Cesium3DTileset.prototype.update = function(frameState) {
+        if (!this._ready) {
```

Comment:
Yeah looking at it now, it's more complicated than it needs to be. It's updated now.